### DISCUSSION TITLE
https://maxkim-j.github.io/posts/coupling/

### DISCUSSION THREAD

**Discussion by giscus** [MaxKim-J/maxkim-j.github.io#98] [Announcements]
# https://maxkim-j.github.io/posts/coupling/

<Reliable software through composite design> 에서 제시한

https://maxkim-j.github.io/posts/coupling/

**Comment by saengmotmi** [comment]
근래 유연하게 결합하기에 대한 생각을 많이 하고 있는데 덕분에 한번 정리하고 갑니다
좋은 글 감사해요!
